lib/main.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:events_flutter/blocs/ThemeSwitch.dart';
```

Comment:
A dart file should never contain Capital letters.
Please change the file name to `theme_switch.dart`